examples/yaml/advanced/02-kueue-integration.yaml

```diff
@@ -0,0 +1,73 @@
+# Advanced Example 2: Kueue Integration
```

Comment:
I'm not sure this is correct actually.

AFAIK TrainJob Integration for Kueue should only require adding a label to the top level TrainJob object.

PodIntegrationOverrides were added for more advancued features for kueue like TopoloyAwareScheduling.